tests/fault_tolerance/cancellation/test_vllm.py

```
@@ -224,13 +228,44 @@ def test_request_cancellation_vllm_aggregated(
+                response = requests.get(
+                    f"http://localhost:{frontend_port}/v1/models", timeout=2
+                )
+                if response.status_code == 200:
+                    if stable_start is None:
+                        stable_start = time.time()
+                    elif time.time() - stable_start >= stable_seconds:
+                        logger.info("Frontend is stable")
+                        return
+                else:
+                    stable_start = None
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)